### ISSUE TITLE
if hide and then show whole sidebar, open files never there.

### ISSUE THREAD

**Issue by thfrlw** [postcasio/tree-view-open-files#12]

**Comment by jstarr** [comment]
I typically use a tall, relatively narrow screen for coding.  Frequently hiding the sidebar gives me just enough room to see whole lines without scrolling.  Then when showing the sidebar to find another open tab, the pane is missing.  The only way to get it back that I have found is to exit atom and restart.

By the way, this is a really useful package.  Because of the narrow width of my screen, if there are more than 6 or 7 tabs open, the names in the tabs are truncated to the point it is difficult to recognize their names.  Having the pane at the top of the sidebar is really convenient.
![image](https://cloud.githubusercontent.com/assets/1629200/6060895/62a501e4-ad10-11e4-8a30-acbc8e43863f.png)

**Comment by soDEVch** [comment]
Also missing the open files pane after toggling the tree view. 
As mentioned above, you can restart atom to get it back, or just disabling and re-enabling the package. 
But it would still be a lot more helpful, if the pane could remain in place when toggling the tree view.

**Comment by mpeebles** [comment]
This also happens when toggling the the side of the tree view.

**Comment by jgonagle** [comment]
Seems the problem is on lines 21-25 in tree-view-open-files.coffee.  The `if` statement is supposed to detect whether the tree view is visible after a `tree-view:toggle` event, but is not doing so.  

Fortunately, the `tree-view:show` event is being handled correctly by this package, so I've added the following to my keymap.cson file to mostly fix the problem. 

```
'.platform-win32, .platform-linux':
    'ctrl-\\': 'tree-view:show'
    'ctrl-|': 'tree-view:toggle'
```

In order to display the tree view with the open files, press ctrl-\.  To hide them, press ctrl-shift-\ (or ctrl-|).  The second command is really a toggle since there's no built in command for tree-view for hiding the tree-view (at least that I'm aware of).

This really should be an easy fix for @postcasio, and I know it almost kept me from using this package the first time I tested it out.
